egs/sprakbanken_no/s5/local/data_prep.py

```diff
@@ -23,10 +23,10 @@
 
 
 # Some filler Speaker IDs to use when Speaker_ID is missing from a json file.
-filler_ids = [385, 647, 908, 397, 909, 143, 271, 145, 910, 19, 911, 912, 790, 913,\
-              24, 25, 26, 536, 796, 919, 31, 162, 163, 164, 546, 683, 684, 46, 177,\
-              51, 179, 53, 446, 447, 448, 449, 196, 70, 843, 845, 464, 340, 344, 345,\
-              605, 609, 610, 611, 866, 749, 241, 242, 243, 244, 245, 507, 915, 255]
+filler_ids = [611, 647, 143, 145, 683, 684, 177, 179, 196, 749, 255, 271,
+			  790, 796, 1016, 843, 845, 340, 344, 345, 866, 385, 397, 915,
+			  919, 922, 923, 924, 925, 926, 927, 928, 929, 930, 931, 932, 
```

Comment:
I have manually added some "filler" speaker ID's since some JSON files dont contain speakerID information. Maybe this is not the proper way to do it.